02week/ticTacToe.js

```diff
@@ -23,26 +30,72 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+/* Run an conditional statement to make rows all equal to the same string
+In rows  make variable for function horizontalWin equal:Row "0"
+=> [[0,0], [0,1], [0,2] and Row "1" => [1,0], [1,1], [1,2]
+and Row"2 => [2,0], [2,1], [2,2]].*/
+const horizontalWin = () => {
+  if ((board[0][0] === board[0][1] && board[0][1] === board[0][2]) || (board[1][0] === board[1][1] && board[1][1] === board[1][2]) || (board[2][0] === board[2][1] &&
```

Comment:
Your win functions seem repetitive, is there a way you can extract and make this more functional?